lockbox-ios/Store/DataStore.swift

```diff
@@ -389,9 +399,8 @@ extension DataStore {
             if !profile.hasAccount() {
                 // first run.
                 self.storageStateSubject.onNext(.Unprepared)
-            } else {
-                self.storageStateSubject.onNext(.Preparing)
             }
+
             self.syncSubject.onNext(.NotSyncable)
```

Comment:
How do we test this state machine? This should probably be the subject of a followup PR.

Reply:
`DataStore.swift` is not tested at all right now :p

There is an issue that sort of addresses this that we'll talk about in tech debt triage next week, but maybe we need another one.